externs/shaka/player.js

```diff
@@ -1029,7 +1030,11 @@ shaka.extern.StreamingConfiguration;
  *   The largest fraction of the estimated bandwidth we should use. We should
  *   downgrade to avoid this.
  * @property {shaka.extern.AdvancedAbrConfiguration} advanced
- *   Advanced ABR configuration.
+ *   Advanced ABR configuration
+ * @property {boolean} qualityRestrictionsElementSize
+ *   If true, limits the quality to the size of the media element.
+ *   Note: The use of ResizeObserver is required for it to work properly.
```

Comment:
Maybe expand this to state what happens when ResizeObserver is not implemented (this field is effectively false).

Reply:
Done!